android/src/com/mapswithme/maps/editor/CuisineAdapter.java

```diff
@@ -66,7 +66,7 @@ public void setFilter(@NonNull String filter)
       return;
     mFilter = filter;
 
-    final String[] filteredKeys = StringUtils.nativeFilterContainsNormalized(Editor.nativeGetCuisines(), filter);
+    final String[] filteredKeys = Editor.nativeFilterCuisinesKeys(Editor.nativeGetCuisines(), filter);
```

Comment:
@biodranik I think maybe instead of 2 native calls I should copy `nativeGetCuisines` implementation into `nativeFilterCuisinesKeys` ?
And this line would look like:

```
Editor.nativeFilterCuisinesKeys(filter);
```

Reply:
It's a good idea if the code can be reused and made simple. An empty filter returns everything, a non-empty filter returns filtered results.

Reply:
Ok, I've simplified `Editor.nativeFilterCuisinesKeys(String)` function. Please review.